spec/features/idv/doc_auth/address_step_spec.rb

```diff
@@ -33,4 +33,12 @@
 
     expect(page).to have_current_path(idv_doc_auth_verify_step)
   end
+
+  it 'sends the user to start doc auth if there is no pii from the document in session' do
+    visit sign_out_url
+    sign_in_and_2fa_user
+    visit idv_address_path
+
+    expect(page).to have_current_path(idv_doc_auth_welcome_step)
```

Comment:
would it be easier to test as a controller spec? then we could inspect instance variables, etc etc